.gitignore

```diff
@@ -41,6 +41,9 @@ output/*/index.html
 docs/_build
 docs/modules.rst
 
+# LevelDB
+evm/db/backends/db
```

Comment:
This doesn't seem appropriate as it will ignore anything found in this directory.  Strikes me that this should be under a different directory than `./evm` since that is the root source dir for this codebase.